src/AzureExtension/DevBox/Helpers/TaskYAMLToCSClasses.cs

```diff
@@ -71,7 +71,9 @@ public class Directives
     {
         public string Description { get; set; } = string.Empty;
 
-        public bool? AllowPrerelease { get; set; }
+        public bool? AllowPrerelease { get; set; }
+
+        public string SecurityContext { get; set; } = string.Empty;
```

Comment:
Is it guaranteed to be set to a valid value? Or is it not serialized if it's empty?
I don't think empty string is a valid value and since we know that "current" and "elevated" are the only valid values and "current" is the default I suggest to initialize this property to "current".